daemon/internal/quota/testhelpers.go

```diff
@@ -12,14 +13,37 @@ import (
 
 // CanTestQuota - checks if xfs prjquota can be tested
 // returns a reason if not
-func CanTestQuota() (string, bool) {
+func CanTestQuota(t *testing.T) (string, bool) {
```

Comment:
Doesn't seem necessary to pass `t *testing.T` just for `t.TempDir()`
```suggestion
func CanTestQuota() (string, bool) {
```

Alternatively you can just omit values and call `t.Skip()` from this function
```suggestion
func RequireSupported(t *testing.T) {
```